chat-plugins/helptickets.js

```diff
@@ -446,7 +446,13 @@ exports.pages = {
 exports.commands = {
 	requesthelp: 'helpticket',
 	helprequest: 'helpticket',
-	report: 'helpticket',
+	report: function (target, room, user, connection) {
+		let message = toId(target);
+		if (message.includes('list') || message.includes('escalate') || message.includes('close') || message.includes('ban') || message.includes('delete')) {
+			return this.parse('/helpticket ' + target);
+		}
+		return this.parse('/helpticket');
+	},
```

Comment:
Can you move this to a different section?

Reply:
it's below the helpticket object now where i assume it should go